composer.json

```
@@ -1,6 +1,10 @@
 {
     "name": "ibexa/polyfill-php82",
     "type": "library",
+    "license": [
+        "GPL-2.0-only",
+        "proprietary"
+    ],
```

This is well defined and required to be:
```suggestion
    "license": "(GPL-2.0-only or proprietary)",
```
See https://github.com/ibexa/core/blob/main/composer.json.

---

I'd counter that array notation is preferred for readability, and suggested by composer:
https://getcomposer.org/doc/04-schema.md#license

---

> I'd counter that array notation is preferred for readability, and suggested by composer: https://getcomposer.org/doc/04-schema.md#license
> 
> Both are, of course, allowed. But I'd argue that array removes the need to external tools to parse a string. WDYT @alongosz ?

Ok, but it would be good then at some point to align it for all other packages, so it's the same (for the purposes of automation).